opentelemetry-sdk/tests/trace/export/test_export.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from opentelemetry.sdk import trace
+from opentelemetry.sdk.trace import export
+
+
+class TestSimpleSpanProcessor(unittest.TestCase):
+    def test_simple_span_processor(self):
+        class MyExporter(export.Exporter):
+            def __init__(self, span_list):
+                self.span_list = span_list
+
+            def export(self, spans: trace.Span) -> export.ExportResult:
+                self.span_list.extend([span.name for span in spans])
+                return export.ExportResult.SUCCESS
+
+        tracer = trace.Tracer()
+
+        spans_names_list = []  # filled by MyExporter
```

Comment:
Instead of this comment, maybe rename the `__init__` parameter to destination and use a keyword argument in the next statement?

Reply:
Done.